src/pages/cli/graphql/authorization-rules.mdx

```diff
@@ -0,0 +1,368 @@
+export const meta = {
+  title: `Authorization rules`,
+  description: `Add authorization rules to your GraphQL schema to control access to your data.`,
+};
+
+Use the `@auth` directive to configure authorization rules for public, sign-in user, per user, and per user group data access. **Authorization rules operate on the deny-by-default principle**. Meaning that if an authorization rule is not specifically configured, it is denied.
+
+```graphql
+type Todo @model @auth(rules: [{ allow: owner }]) {
+  content: String
+}
+```
+
+In the example above, each signed-in user, or also known as "owner", of a Todo can create, read, update, and delete their own Todos. 
+
+Amplify also allows you to restrict the allowed operations, combine multiple authorization rules, and apply fine-grained field-level authorization. 
+
+``` graphql
+type Todo @model @auth(rules: [
+  { allow: public, operations: [read]}
+  { allow: owner }) {
+  content: String
+}
+```
+
+In the example above, everyone (`public`) can read every Todo but owner (authenticated users) can create, read, update, and delete their own Todos.
+
+### Global authorization rule (only for getting started)
+
+To help you get started, there's a global authorization rule defined when you create a new GraphQL schema. For production environments, remove the global authorization rule and apply rules on each model instead.
+
+```graphql
+input AMPLIFY { globalAuthRule: AuthRule = { allow: public } }
+```
+
+The global authorization rule (in this case `{ allow: public }` - allows anyone to create, read, update, and delete) is applied to every data model in the GraphQL schema. 
+
+<Callout warning>
+
+Currently, only `{ allow: public }` is supported as a global authorization rule.
+
+</Callout>
+
+## Authorization strategies
+
+Use the guide below to select the correct authorization strategy for your use case:
+
+| **Recommended use case** | **Strategy** | **Provider** |
+|---|---|---|
+| Public data access. Anyone with the AppSync API Key should have access. | [`public`]() | `apiKey` |
+| Recommended for production environment's public data access. Anyone signed in or not (verified by IAM) should have access. | [`public`]() | `iam` |
```

Comment:
Should this include which role is being used here? From the CLI's POV this is referring to unauthenticated access vended by the identity pool. Android doc example on unauthenticated access: https://docs.amplify.aws/sdk/auth/guest-access/q/platform/android/

Reply:
I've added this now into the details of the section.